data_store/cli.py

```diff
@@ -18,148 +23,276 @@
+
+                    # Go to next email
+                    continue
+
+                # Try to get account by email
+                try:
+                    response = requests.get(
+                        f"{Config.ACCOUNT_STORE_API_HOST}{Config.ACCOUNTS_ENDPOINT}",
+                        headers={"Content-Type": "application/json"},
+                        params={"email_address": email},
+                    )
+
+                    response.raise_for_status()
+
+                    account = response.json()
+                except RequestException as e:
+                    if e.response.status_code != 404:
+                        result["errors"][email] = e.response.text
+                        click.echo(f"Get account with {email} error: {e.response.text}")
+
+                        # Go to next email
+                        continue
+
+                    # If account not found, set account to None and we create it
+                    account = None
+
+                if account is None:
+                    click.echo(f"Creating account for {email}")
+
+                    # Create account
+                    try:
+                        response = requests.post(
+                            f"{Config.ACCOUNT_STORE_API_HOST}{Config.ACCOUNTS_ENDPOINT}",
+                            headers={"Content-Type": "application/json"},
+                            json={"email_address": email},
+                        )
+
+                        response.raise_for_status()
+
+                        result["created"] += 1
+                        account = response.json()
+                    except requests.exceptions.RequestException as e:
+                        result["errors"][email] = e.response.text
+                        click.echo(f"Create account for {email} error: {e.response.text}")
+
+                        # Go to next email
+                        continue
+
+                # Update account with roles
+                data = {
+                    "email_address": email,
+                    "azure_ad_subject_id": account.get("azure_ad_subject_id"),
+                    "full_name": full_name,
+                    "roles": roles,
+                }
+
+                account_id = account.get("account_id")
+
+                try:
+                    response = requests.put(
+                        Config.ACCOUNT_STORE_API_HOST + Config.ACCOUNT_ENDPOINT.format(account_id=account_id),
+                        headers={"Content-Type": "application/json"},
+                        json={k: v for k, v in data.items() if v is not None},
+                    )
+
+                    response.raise_for_status()
+
+                    result["updated"] += 1
+                except requests.exceptions.RequestException as e:
+                    result["errors"][email] = e.response.text
+                    click.echo(f"Update account for {email} error: {e.response.text}")
```

Comment:
We're missing a `continue` here that the other `except` blocks have

Reply:
Fixed! :+1: